homeassistant/components/habitica/services.yaml

```diff
@@ -33,7 +33,262 @@ cast_skill:
             - "fireball"
           mode: dropdown
           translation_key: "skill_select"
-    task:
+    task: &task
       required: true
       selector:
         text:
+update_todo:
+  fields:
+    config_entry: *config_entry
+    task: *task
+    rename: &rename
+      required: false
+      selector:
+        text:
+    description: &description
+      required: false
+      selector:
+        text:
+          multiline: true
+    checklist_options: &checklist_options
+      collapsed: true
+      fields:
+        add_checklist_item:
+          required: false
+          selector:
+            text:
+              multiple: true
+        remove_checklist_item:
+          required: false
+          selector:
+            text:
+              multiple: true
+        score_checklist_item:
+          required: false
+          selector:
+            text:
+              multiple: true
+        unscore_checklist_item:
+          required: false
+          selector:
+            text:
+              multiple: true
+    priority: &priority
+      required: false
+      selector:
+        select:
+          options:
+            - "trivial"
+            - "easy"
+            - "medium"
+            - "hard"
+          mode: dropdown
+          translation_key: "priority"
+    duedate_options:
+      collapsed: true
+      fields:
+        date:
+          required: false
+          selector:
+            date:
+    clear_date:
+      required: false
+      selector:
+        constant:
+          value: true
+          label: ""
```

Comment:
The constant selectors cannot be nested below a section due to a bug in the frontend, will be updated when it is fixed
- https://github.com/home-assistant/frontend/issues/22297
- https://github.com/home-assistant/frontend/pull/22299